src/2048-expert/index.js

```diff
@@ -213,6 +213,7 @@ function keyUpLeft() {
     moveLeft()
     generateNewTile()
```

Comment:
Due to the nature of the game, everywhere that you use `generateNewTile()`, you use it twice in a row. So, maybe you could have another function called `generateTwoTiles()` to reduce a bit of repetition.

Reply:
Good idea